### DISCUSSION TITLE
How exactly does "setActorX", "setActorY", etc. work?

### DISCUSSION THREAD

**Discussion by redstonemaster60** [Leather128/LeathersFunkinEngine#242] [Q&A] [answered]
I haven't been able to figure this out for a bit, but I have a few general questions about it.
1. Is there any list of what things are actors and if so, where is it?
2. If there's no list, what is generally an "actor"? I'm assuming that the arrows, characters and stage are but is there anything else or just specific groups of things that are all actors?
I know this might be obvious but if you know it please let me know, thanks

**Comment by Leather128** [comment, marked as answer]
1. There is no list, basically just `boyfriend`, `girlfriend`, and `dad` (+ numbers from 0-song's key count times 2 minus 1 for strums), and then add any stage sprites as `undefinedSprite[index in stage list]` or the name of the object if specified.
2. An actor is just any sprite at all, and there are no specific groups really, just sprites lol.
